### ISSUE TITLE
Neural Network Example | Zerohertz

### ISSUE THREAD

**Issue by Zerohertz** [Zerohertz/zerohertz.github.io#191]
https://zerohertz.github.io/neural-network-example/ 

 Import Packages1234>>> import torch>>> import numpy as np>>> import matplotlib.pyplot as plt>>> from torch.autograd import Variable  TensorScala (0-Dim Tensor)12345
